src/openlcb/DccAccyConsumer.hxx

```diff
@@ -67,7 +65,8 @@ public:
         memset(isStateKnown_, 0, sizeof(isStateKnown_));
     }
 
-    ~DccAccyConsumer()
+    /// Destructor.
+    ~DccAccyConsumerBase()
     {
         EventRegistry::instance()->unregister_handler(this);
     }
```

Comment:
we should probably add a public: after this so that the virtual public interafce methods of the simpleeventhandler do not have their visibility changed. Then another protected: before the class variables.

Reply:
I don't agree that this is required. As long as the virtual methods are declared public in the SimpleEventHandler class, they can always be called on that object type. We don't want anyone calling these methods on the DccAccyConsumerBase object type, so they should not be public.